test/verify/check-services

```diff
@@ -219,7 +219,7 @@ Unit=test.service
         self.check_service_details(["Disabled"], ["Disallow running (mask)"], False)
         self.toggle_onoff()
         self.check_service_details(["Failed to start", "Automatically starts"], ["Start", "Reload", "Disallow running (mask)"], True)
-        b.wait_visible(".action-button:contains('Start Service')")
+        b.click(".action-button:contains('Start Service')")
```

Comment:
If that actually failed before, could the test check the result, i. e. the status flipping to "running"?

Reply:
I think the service is not going to run, it will just fail again. The problem this fixes was that once you clicked on the button you got oops, which would fail the whole test. Thats why I changed it from `wait_visible` to `click` as `click` also checks `visible` but also it triggers the callback.

Reply:
Makes sense, thanks!